server/routes/unauthenticatedRoutes.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express'
+
+import paths from '../constants/paths'
+
+export default function routes(): Router {
```

Comment:
Haven't bothered with individual controllers for the various 'accessibility', 'privacy' etc pages as they're so simple.